Formula/kde-attica.rb

```diff
@@ -0,0 +1,49 @@
+class KdeAttica < Formula
+  desc "Open Collaboration Service client library"
+  homepage "https://api.kde.org/frameworks/attica/html/index.html"
+  url "https://download.kde.org/stable/frameworks/5.79/attica-5.79.0.tar.xz"
+  sha256 "8af244b41f08448ea3693e9f7d9b50de9df76b416016cd1143dfc581dd65d9dc"
+  head "https://invent.kde.org/frameworks/attica.git"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "doxygen" => :build
+  depends_on "graphviz" => :build
+  depends_on "kde-extra-cmake-modules" => [:build, :test]
+  depends_on "ninja" => :build
+
+  depends_on "qt@5"
+
+  def install
+    args = std_cmake_args
+    args << "-D BUILD_QCH=ON"
+    args << "-D BUILD_TESTING=OFF"
+    args << "-D BUILD_TESTS=OFF"
+    args << "-D BUILD_UNITTESTS=OFF"
+    args << "-D CMAKE_INSTALL_BUNDLEDIR=#{bin}"
+    args << "-D KDE_INSTALL_LIBDIR=lib"
+    args << "-D KDE_INSTALL_PLUGINDIR=lib/qt5/plugins"
+    args << "-D KDE_INSTALL_QMLDIR=lib/qt5/qml"
+    args << "-D KDE_INSTALL_QTPLUGINDIR=lib/qt5/plugins"
+
+    mkdir "build" do
+      system "cmake", "..", "-G", "Ninja", *args
+      system "ninja"
+      system "ninja", "install"
+      prefix.install "install_manifest.txt"
+    end
+  end
+
+  test do
+    (testpath/"CMakeLists.txt").write <<~EOS
+      project(test_KF5Attica)
+      find_package(Qt5 REQUIRED Core Network)
+      find_package(KF5Attica REQUIRED)
+    EOS
+
+    args = std_cmake_args
+    args << "-DQt5_DIR=#{Formula["qt@5"].opt_prefix/"lib/cmake/Qt5"}"
+
+    system "cmake", testpath.to_s, *args
+    system "make"
+  end
```

Comment:
Doesn't this just test of cmake can find it? Instead of functions?

Reply:
Find it and load the target, but not whether the libs load properly. I'll try to add more complex tests to these formulas

Reply:
Actually, @SMillerDev, is that necessary? The cookbook uses exactly this type of test as an example of a good one:

```
See cmake for an example of a formula with a good test. The formula writes a basic CMakeLists.txt file into the test directory then calls CMake to generate Makefiles. This test checks that CMake doesn’t e.g. segfault during basic operation.
```

The cmake test is defined as:

```
  test do
    (testpath/"CMakeLists.txt").write("find_package(Ruby)")
    system bin/"cmake", "."
  end
```